project-keeper/src/main/java/com/exasol/projectkeeper/validators/changesfile/ChangesFileSection.java

```diff
@@ -38,6 +41,20 @@ public List<String> getContent() {
         return this.content;
     }
 
+    Stream<FixedIssue> getFixedIssues() {
```

Comment:
This looks a bit strange. 
1. Method `findFixedIssues()` will return either an empty stream or a stream of a single issue, but not multiple. 
2. nesting streams and then using `flatMap()` seems a bit around-the-corner to me.

Reply:
I tried it with Optional, please check if it is better now :)